website/docs/api/plugins/plugin-content-docs.md

```diff
@@ -55,6 +55,8 @@ Accepted fields:
 | `beforeDefaultRehypePlugins` | `any[]` | `[]` | Custom Rehype plugins passed to MDX before the default Docusaurus Rehype plugins. |
 | `showLastUpdateAuthor` | `boolean` | `false` | Whether to display the author who last updated the doc. |
 | `showLastUpdateTime` | `boolean` | `false` | Whether to display the last date the doc was updated. |
+| `showCreateAuthor` | `boolean` | `false` | Whether to display the author who created the doc. |
+| `showCreateTime` | `boolean` | `false` | Whether to display the date the doc was created. |
```

Comment:
It's been close to a year and I don't even wholly recall my motivation for this feature...😅 Do we want it as an out-of-the-box feature, or only provide the metadata and wait for users who swizzled it to discover it? (Not a nice thing to do) I don't have a compelling use-case apart from telling two sets of documentation apart (which one is "legacy" and which one is "current"). We could use also use versioning for that purpose, though. I would need to think of a better use-case. Eventually we need to address https://github.com/facebook/docusaurus/issues/6218, which requires _much more_ metadata than this.

Reply:
I can’t think of a very good use-case either. I thought there was a reason that I didn’t think of 😅. Should I continue working on this feature? I am happy to close the pull request if there is no strong reason for keeping it.

Reply:
Given it's not too complex, opt-in, and may give additional flexibility by exposing the creation date, I think we can keep working on this.

The other possibility for users would be to use another flexible API (like `createFrontMatter`? https://github.com/facebook/docusaurus/issues/5568) so that they can create this `created` metadata themselves through some callback.

Reply:
Got it! I'll finish this up then look into the other metadata APIs.

Reply:
> then look into the other metadata APIs.

what do you mean? 😅

Reply:
> then look into the other metadata APIs.

Never mind, I was misinterpreting something. Sorry for confusion.